src/network.js

```diff
@@ -1,3 +1,5 @@
 export const NETWORK_BYTE = 'M'.charCodeAt(0);
 export const NODE_IP = '/api';
 export const VSYS_RATE = 'https://vsysrate.com/api/superNodesDetail'
+export const CERTIFICATED_TOKEN = 'https://explorer.v.systems/api/tokens'
```

Comment:
Only keep `EXPLORER` config here. `CERTIFICATED_TOKEN` changes to '/api/tokens' and put into `constants.js`. 
In `constants.js`, also change `TX_EXPLORER` to `TX_URL` and value change to '/transactions/'.